piccolo_api/crud/endpoints.py

```diff
@@ -446,6 +455,11 @@ def _split_params(params: t.Dict[str, t.Any]) -> Params:
                     response.page_size = page_size
                 continue
 
+            if key == "__visible_fields":
+                visible_fields = str(value)
```

Comment:
This might not be needed - is it already a string?

Reply:
Yes it's a string. I'll change this